proplot/tests/test_journals.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+import proplot as plot
+from proplot.subplots import JOURNAL_SPECS
+
+
+# Loop through all available journals.
+@pytest.mark.parametrize('journal', JOURNAL_SPECS.keys())
+def test_journal_subplots(journal):
+    """Tests that subplots can be generated with journal specifications."""
+    f, axs = plot.subplots(journal=journal)
```

Comment:
Could even add `assert` clause here to check that the figure with matches the expected width from the journals table.

Reply:
Thanks for the help! Yeah we could do that in a broader test of `_subplots_geometry` -- this is the engine that calculates figure geometry based on the "reference subplot" aspect ratio and whether width, height, axwidth, and/or axheight were input. When you specify a journal name, the relevant dimensions are just passed to the algorithm.